.github/workflows/test.yml

```diff
@@ -78,3 +78,22 @@ jobs:
               uses: codecov/codecov-action@v1.0.10
               with: 
                 yml: ./codecov.yml
+
+    rpcTest:
+        name: RPC Generator Tests
+        runs-on: ubuntu-latest
+        strategy:
+            fail-fast: false
+        steps:
+            - name: Checkout repository
+              uses: actions/checkout@v2.3.1
+              with:
+                  submodules: true
+            - name: Current Directory
+              run: ls
+            - name: Install dependencies
+              run: |
+                python3 -m pip install -r generator/requirements.txt
```

Comment:
Do you need to do the pipe or can you just put the bottom line on the same line as the `run` command like other test in the file above

Reply:
I can remove the pipe. It was used before to combine commands, now there is no need to it.